### ISSUE TITLE
protobuf-java 3.12.3 not published

### ISSUE THREAD

**Issue by augi** [protocolbuffers/protobuf#7600]
The latest version listed in Maven Central is 3.12.2 but there is two days old GitHub release of 3.12.3.

https://search.maven.org/artifact/com.google.protobuf/protobuf-java

**Comment by azlux** [comment]
Same on PyPI https://pypi.org/project/protobuf/#history

**Comment by anderscarling** [comment]
Same on [cocoapods](https://cocoapods.org/pods/Protobuf), [rubygems](https://cocoapods.org/pods/Protobuf) and [npm](https://www.npmjs.com/package/google-protobuf).

**Comment by tinder-aminghadersohi** [comment]
Any update on this please?

**Comment by AustinShalit** [comment]
@haberman Do you need to hit the release button on Sonatype again?

https://github.com/protocolbuffers/protobuf/issues/7473#issuecomment-624976285

**Comment by AustinShalit** [comment]
@acozzette Do you have the ability to cut a release?

**Comment by acozzette** [comment]
Yes, I am going to just do a 3.12.4 release, because 3.12.3 was also mistakenly tagged at the wrong commit (see #7632).

**Comment by AustinShalit** [comment]
@acozzette Do you have a timeframe for that release? I just want to know how much I should work around the version (ala https://xkcd.com/1319/)

**Comment by acozzette** [comment]
I'm publishing the 3.12.4 artifacts now and should be done before the end of the day.

**Comment by AustinShalit** [comment]
Awesome! Thank you so much!

**Comment by AustinShalit** [comment]
Hi @acozzette, Just a polite reminder to push the java artifacts to maven and python artifacts to pypi 🙂

**Comment by acozzette** [comment]
@AustinShalit All the artifacts for 3.12.4 should already be there, but let me know if I missed something.
